### PR TITLE
Add pattern filter for suite selector

### PR DESCRIPTION
**Pull request by missedone**
Hi
as we have a lot of testng xml suite which caused a quite long list in the suite selector dialog, so i add a pattern filter text so that we can quickly filter out the expected suites name

### CODE REVIEW

File: src/main/org/testng/eclipse/launch/SuiteSelector2.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
   SuiteSelector2(TestNGMainTab callback, Composite comp) {
     super(callback, comp, LaunchType.SUITE, "TestNGMainTab.label.suiteTest",
         "CheckBoxTable.suites.title");
-    setTextEditable(false); // no manual edit
+    setTextEditable(true); // no manual edit
```
Comment (missedone) [inline src/main/org/testng/eclipse/launch/SuiteSelector2.java]: make it editable to be able to modify the order easier

Comment (JnRouvignac) [inline src/main/org/testng/eclipse/launch/SuiteSelector2.java]: You should remove the associated comment - which should never have been there in the first place and is now outdated.

Comment (missedone) [inline src/main/org/testng/eclipse/launch/SuiteSelector2.java]: yes, indeed, just checked in the changes, see the updated pull request:
https://github.com/cbeust/testng-eclipse/pull/130/files

thanks

  Thanks & Best Regards!

                  ///
                 (. .)
  --------ooO--(_)--Ooo--------
  |           Nick Tan           |
  ------------------------------------

**Comment by cbeust** [comment]
Thanks!

**Pull request merged**